### DISCUSSION TITLE
Button Toggle - set toggle-color and toggle-text-color based on selected option

### DISCUSSION THREAD

**Discussion by hamidhosseini** [quasarframework/quasar#13777] [General - Components / Directives / etc]
Can someone help me figure this out? 
I have two teams, one is orange, the other one is blue. I want blue to be selected by default and when orange is selected, the toggle color change to orange.
For this, I need the selected option but how can I know which option is selected?

![image](https://user-images.githubusercontent.com/35292387/175550720-5a8e3eb4-2f33-4d88-82ed-d6fbb55125e6.png)

**Comment by MilosPaunovic** [comment]
[https://codepen.io/milospaunovic/pen/NWyQMRo?editors=1010](https://codepen.io/milospaunovic/pen/NWyQMRo?editors=1010)
